packages/@okta/vuepress-site/docs/guides/add-an-external-idp/create-an-app-at-idp/oktatookta/appatidp.md

```diff
@@ -1,22 +1,17 @@
 When you are configuring federation between two Okta orgs using OpenID Connect:
 
-1. At the Okta org that represents the Identity Provider, select **Applications** and then click **Add Application**.
-
-2. You need a trusted client, so select **Web** as the platform. OpenID Connect is the sign-in method by default.
-
-3. Click **Next**.
-
-4. Enter a name for your application.
-
-5. In the **Login Redirect URIs** box, enter the redirect URI. The redirect URI:
+1. In the Admin Console for the Okta org that represents the Identity Provider, go to **Applications** > **Applications**.
+1. Click **Create App Integration**.
+1. On the Create a new app integration page, select **OIDC - OpenID Connect** as the **Sign-in method**. You need a trusted client, so select **Web Application** as the **Application type**.
+1. Click **Next**.
+1. Enter a name for your application.
+1. In the **Sign-in redirect URIs** box, enter the redirect URI. The redirect URI:
     * Is constructed using your Okta org subdomain and the callback endpoint  
     For example, if your Okta subdomain is called `company`, then the URI would be: `https://company.okta.com/oauth2/v1/authorize/callback`. If you have configured a custom domain in your Okta Org, use that value to construct your redirect URI, such as `https://login.company.com/oauth2/v1/authorize/callback`.
     * Is where the IdP returns the authentication response (the access token and the ID token)
     * Needs to be a secure domain that you own
     * Should match the redirect URI sent in the authorize request from the client
 
-6. Assign a group or leave the **Everyone** default. Be sure to verify that the users you want to have access are assigned to the group that you select.
-
-7. Click **Done**.
-
-8. Copy the **Client ID** and **Client secret** from the **Client Credentials** section and paste into a text editor. You need these when you configure this Identity Provider in your other Okta org in the next section.
+1. Assign a group or leave the **Everyone** default. Be sure to verify that the users you want to have access are assigned to the group that you select.
+1. Click **Save**.
+1. Copy the **Client ID** and **Client secret** from the **Client Credentials** section and paste into a text editor. You need these when you configure this Identity Provider in your other Okta org in the next section.
```

Comment:
Client ID and Client secret:
  Client ID and Client Secret (initial uppercase for "secret"?)
  -OR-
  client ID and client secret (all lowercase?)

Reply:
@christinechoy-okta - The field name in the UI is **Client service**, so we have to reflect what is in the UI. 
<img width="761" alt="Screen Shot 2021-05-17 at 9 12 56 AM" src="https://user-images.githubusercontent.com/45509385/118512505-17a49380-b6f0-11eb-98e3-0fb7938b470e.png">

Reply:
No change - mirror the UI